storage/replica.go

```diff
@@ -241,11 +247,11 @@ type Replica struct {
 	// raftMu protects Raft processing the replica.
 	//
 	// Locking notes: Replica.raftMu < Replica.mu
-	raftMu syncutil.Mutex
+	raftMu sync.Locker
```

Comment:
No, it's legal to copy a `sync.Locker` (which is an interface). In other words, this code passes:

```go
type R struct {
       	sync.Locker
}

func main() {
       	var s, t R
       	s = t
       	_ = s
}
```